
class ResourceUnavailableException(Exception):
    pass


class ResourceExistsException(Exception):
    pass